src/node/waku_node.py

```diff
@@ -12,6 +15,12 @@
 logger = get_custom_logger(__name__)
 
 
+def select_private_key(prv_keys, key_id):
+    for key in prv_keys:
```

Comment:
can we return a error or something meaningfull it there is not key or the keys does not end with key_id?
right now it returns None to eth_private_key and the code will continue